### PR TITLE
feat: add bundle analyzer

### PR DESCRIPTION
**Pull request by vpicone**
test: 
- run `build:analyze`
- check out `/guides/npm-scripts` for new content

### CODE REVIEW

File: packages/example/src/data/nav-items.yaml [hunk 1]
```diff
@@ -22,6 +22,8 @@
       path: /guides/styling
     - title: Navigation
       path: /guides/navigation/sidebar
+    - title: Navigation
```
Comment (sstrubberg) [inline packages/example/src/data/nav-items.yaml]: rename to scripts or NPM scripts or something like that..
image

Comment (vpicone) [reply packages/example/src/data/nav-items.yaml]: I hate how they don't capitalize 😢

**Comment by sstrubberg** [comment]
after i run `yarn build:analyze`, it runs, but then it just stops. 127.0.0.1:3001 is just a "Page can not be displayed"

**Comment by vpicone** [comment]
@sstrubberg should be good now

**Pull request merged**